### PR TITLE
Update from discovery

### LINKED ISSUE #276: Update example Discovery pages with missing page components
Little things are missing here and there, h4s into h3s, missing links, etc.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes #276 and #274

### CODE REVIEW

**Comment by ricardoom** [comment]
taking a look now

File: sass/_toolkit.scss [hunk 1]
```diff
@@ -464,6 +464,10 @@
 
 .nypl-results-item {
   @include results-item;
+
```
Comment (ricardoom) [inline sass/_toolkit.scss]: Is there a way to put this in one of the partials? _toolkit convention is that this rule would go into the mixin...

Comment (EdwinGuzman) [reply sass/_toolkit.scss]: Do you mean a separate file partial? Or a mixin partial?

Comment (ricardoom) [reply sass/_toolkit.scss]: yeah, could you put the class and rule in the mixin `results-item`?

**Comment by EdwinGuzman** [comment]
@ricardogalvez updated!

**Comment by EdwinGuzman** [comment]
Looks good @ricardogalvez ?

**Comment by EdwinGuzman** [comment]
I wish.

**Pull request merged**